tests/frontend/callbacks/test_variables_callbacks.py

```diff
@@ -310,76 +310,131 @@ def test_populate_variables_workspace_filter_variables(
     )
 
 
-def test_update_variables_values_from_dataset_values(metadata: DataDocMetadata):
+@pytest.mark.parametrize(
+    (
+        "dataset_value",
+        "dataset_identifier",
+        "variable_identifier",
+    ),
+    [
+        (
+            "STATUS",
+            DatasetIdentifiers.TEMPORALITY_TYPE,
+            VariableIdentifiers.TEMPORALITY_TYPE,
+        ),
+        (
+            "01",
+            DatasetIdentifiers.DATA_SOURCE,
+            VariableIdentifiers.DATA_SOURCE,
+        ),
+        (
+            "2009-01-02",
+            DatasetIdentifiers.CONTAINS_DATA_FROM,
+            VariableIdentifiers.CONTAINS_DATA_FROM,
+        ),
+        (
+            "2021-08-10",
+            DatasetIdentifiers.CONTAINS_DATA_UNTIL,
+            VariableIdentifiers.CONTAINS_DATA_UNTIL,
+        ),
+    ],
+)
+def test_update_variables_values_from_dataset_values(
```

Comment:
```suggestion
def test_variables_values_inherit_dataset_values(
```

I think this is a clearer test name, do you agree?